shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js

```diff
@@ -516,69 +512,46 @@ const serveCaseToIrsInteractor = async (
   if (caseEntity.orderForFilingFee) {
     const { orderForFilingFee } = SYSTEM_GENERATED_DOCUMENT_TYPES;
 
-    const content = replaceBracketed(
-      orderForFilingFee.content,
-      todayPlus60,
-      todayPlus60, // since there are 2 instances of the date, replace a 2nd time
-    );
-
-    await applicationContext
-      .getUseCaseHelpers()
-      .addDocketEntryForSystemGeneratedOrder({
-        applicationContext,
-        caseEntity,
-        systemGeneratedDocument: {
-          ...orderForFilingFee,
-          content,
-        },
-      });
+    await generateDraftDocument({
+      applicationContext,
+      caseEntity,
+      document: orderForFilingFee,
+      replacements: [todayPlus60, todayPlus60],
+    });
   }
 
   if (caseEntity.orderForAmendedPetition) {
     const { orderForAmendedPetition } = SYSTEM_GENERATED_DOCUMENT_TYPES;
 
-    const petitionDocument = caseEntity.docketEntries.find(
-      doc => doc.documentType === INITIAL_DOCUMENT_TYPES.petition.documentType,
-    );
-
-    const content = replaceBracketed(
-      orderForAmendedPetition.content,
-      formatDateString(petitionDocument.servedAt, FORMATS.MONTH_DAY_YEAR),
-      todayPlus60,
-      todayPlus60,
-    );
-
-    await applicationContext
-      .getUseCaseHelpers()
-      .addDocketEntryForSystemGeneratedOrder({
-        applicationContext,
-        caseEntity,
-        systemGeneratedDocument: {
-          ...orderForAmendedPetition,
-          content,
-        },
-      });
+    await generateDraftDocument({
```

Comment:
Should this method be put onto applicationContext.getUseCaseHelpers? That way this interactor tests can verify it was called and the actual implementation can be unit tested.

Reply:
@rachelschneiderman, my thinking was that since we don't need it anywhere else right now, it's a concern only of this use case. So the unit tests of the interactor are testing it. And they test it well because I can make the tests fail as expected by changing the `generateDraftDocument` code.

Reply:
Ok, I'm alright with not adding it to app context. I think there should still be unit tests around that new use case though.